cmd/limactl/start-at-login.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"errors"
+	"os"
+	"runtime"
+
+	"github.com/lima-vm/lima/pkg/autostart"
+	"github.com/lima-vm/lima/pkg/store"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func startAtLoginCommand() *cobra.Command {
+	startAtLoginCommand := &cobra.Command{
+		Use:               "start-at-login INSTANCE",
+		Short:             "Register/Unregister the unit file for instance",
+		Args:              WrapArgsError(cobra.MaximumNArgs(1)),
+		RunE:              startAtLoginAction,
+		ValidArgsFunction: startAtLoginComplete,
+		GroupID:           advancedCommand,
+	}
+
+	startAtLoginCommand.Flags().Bool(
+		"enabled", true,
+		"the "+map[string]string{"darwin": "launchd", "linux": "systemd"}[runtime.GOOS]+" unit file for instance",
+	)
+
+	return startAtLoginCommand
+}
+
+func startAtLoginAction(cmd *cobra.Command, args []string) error {
+	instName := DefaultInstanceName
+	if len(args) > 0 {
+		instName = args[0]
+	}
+
+	inst, err := store.Inspect(instName)
+	if err != nil {
+		if errors.Is(err, os.ErrNotExist) {
+			logrus.Infof("Instance %q not found", instName)
+			return nil
+		}
+		return err
+	}
+
+	flags := cmd.Flags()
+	startAtLogin, err := flags.GetBool("enabled")
+	if err != nil {
+		return err
+	}
+	if startAtLogin {
+		if err := autostart.CreateStartAtLoginEntry(runtime.GOOS, inst.Name, inst.Dir); err != nil {
+			logrus.WithError(err)
+		} else {
+			logrus.Infof("The autostart file (%q) has been created", autostart.GetStartAtLoginEntryPath(runtime.GOOS, inst.Name))
+		}
+	} else {
+		deleted, err := autostart.DeleteStartAtLoginEntry(runtime.GOOS, instName)
+		if err != nil {
+			logrus.WithError(err)
+		} else if deleted {
+			logrus.Infof("The autostart file (%q) has been deleted", autostart.GetStartAtLoginEntryPath(runtime.GOOS, instName))
+		}
```

Comment:
I think we should have another `else` clause here and log that the autostart file didn't exist, so the user doesn't worry about why there is no feedback:

```console
$ l start-at-login alpine
INFO[0000] The autostart file ("/Users/jan/Library/LaunchAgents/io.lima-vm.autostart.alpine.plist") has been created
$ l start-at-login alpine --enabled=false
INFO[0000] The autostart file ("/Users/jan/Library/LaunchAgents/io.lima-vm.autostart.alpine.plist") has been deleted
$ l start-at-login alpine --enabled=false
```